canonical/jaas-dashboard
# WD-1199 - Create app slice

## Done

Moved app actions, reducers and thunk to the store dir.
Updated actions and thunks to use the redux toolkit helpers.
Added missing tests.

## QA

- Pull code
- Run `dotrun clean && dotrun serve`
- Open http://0.0.0.0:8036/
- Check that the UI loads and displays models and controllers.

## Details

https://warthogs.atlassian.net/browse/WD-1199
